cob_bms_driver/src/power_state_aggregator.py

```diff
@@ -20,13 +20,15 @@
 import warnings
 from std_msgs.msg import Float64
 from cob_msgs.msg import PowerState
+from sensor_msgs.msg import BatteryState
 
 class PowerStateAggregator():
 
     def __init__(self):
         # get parameters
-        self.current_buffer_size = rospy.get_param('~current_buffer_size', 11) # use odd number to prevent 0.0 mean
+        self.current_buffer_size = rospy.get_param('~current_buffer_size', 901) # approx. 60sec, topic rate 15Hz, use odd number to prevent 0.0 mean
```

Comment:
Not sure if the buffer_size is now to big. I've used a buffer_size of 11 on the sps_bridge as well. Did the old buffer size made Problems?